Rm remaining instances of OperatorSources (#26277)
openshift/openshift-docs: modules/olm-operatorhub-architecture.adoc

The OperatorHub UI component is driven by the Marketplace Operator by default on
{product-title} in the `openshift-marketplace` namespace.

The Marketplace Operator manages the OperatorHub custom resource
definition (CRD).

> 

Reviewer: ```suggestion
The Marketplace operator manages the OperatorHub CustomResource named `cluster` 
```
There can be other OperatorHub CRs too, but marketplace won't respond to changes in them. The OperatorHub CRD is not managed by marketplace

[id="olm-operatorhub-arch-operatorhub_crd_{context}"]
== OperatorHub CRD

> OperatorHub uses CatalogSource objects to source Operator catalogs. You can

Reviewer: ```suggestion
OperatorHub can be used to manage default CatalogSource objects. You can
```

> modify the OperatorHub CRD to change the state of the default catalogs provided

Reviewer 2: ```suggestion
modify the OperatorHub CRD to change the state of the default catalogs that are provided
```
